johnsonandjohnson/Bodiless-JS
# feat(layout): fix rtl flow container.

## Changes:
- Fix flow container on rtl pages.
- Allow for a rtl flow container.
## Test Instructions:
- Visit /flow-container-rtl
- add a component 
- note the drag handle on the left side working as expected.

## Related Issue:
N/A
